lib/messaging/controls/write.rb

```diff
@@ -8,7 +8,7 @@ def self.example
       class Example
         include Messaging::Write
 
-        virtual :configure
+        def configure(*); end
```

Comment:
We don't normally implement one line methods like this. I believe the previous implementation may have had the correct idea, so `template_method(:configure)` is acceptable here (assuming it works).

Otherwise, this should be:

```ruby
def configure(*)
end
```

Reply:
Messaging::Write already defines `template_method! :configure`, so another `template_method :configure` is ignored. The only way to override template method which raises error is to define empty method.